src/plugins/bodhi.c

```diff
@@ -271,6 +270,36 @@ static void bodhi_print_errors_from_json(json_object *json)
     return;
 }
 
+/**
+ * Parses only name of rpm package
+ */
+static int parse_pkginfo_name(const char *pkg_full_name, char **name)
```

Comment:
Good catch! I have added the parse `nevra` function while I should have add a parse `nvr` function. How comes the tests didn't reveal this issue? Can you please analyze it more and update the tests to do better job?

I don's see any problem in fixing this issue and I don't believe it can cause any problems as the only production is Fedora Bodhi.